src/WebJobs.Script.WebHost/Security/SimpleWebTokenHelper.cs

```diff
@@ -80,6 +88,12 @@ public static string Decrypt(byte[] encryptionKey, string value)
                         binaryWriter.Write(data, 0, data.Length);
                     }
 
+                    var input = ms.ToArray();
```

Comment:
We're also doing a ToArray below - let's not duplicate this.

Reply:
Changed to use the input variable in the return